bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2273,7 +2268,8 @@ void ReplicaImp::onFastPathCommitVerifyCombinedSigResult(SeqNum seqNumber,
   auto span = concordUtils::startChildSpanFromContext(fcp->spanContext<std::remove_pointer<decltype(fcp)>::type>(),
                                                       "bft_execute_committed_reqs");
 
-  metric_total_committed_sn_++;
+  auto commitPath = seqNumInfo.getFastPathSelfPartialCommitProofMsg()->commitPath();
```

Comment:
The commit path is already known as a function argument of onFastPathCommitVerifyCombinedSigResult

Reply:
Changed to use the parameter